ignite/services/plugin/template/main.go.plush

```diff
@@ -3,118 +3,47 @@ package main
 import (
 	"context"
 	"fmt"
-	"path/filepath"
 
 	hplugin "github.com/hashicorp/go-plugin"
 
-	"github.com/ignite/cli/v28/ignite/services/chain"
 	"github.com/ignite/cli/v28/ignite/services/plugin"
+	"<%= ModuleName %>/cmd"
 )
 
 type app struct{}
 
-func (app) Manifest(ctx context.Context) (*plugin.Manifest, error) {
+func (app) Manifest(_ context.Context) (*plugin.Manifest, error) {
 	return &plugin.Manifest{
-		Name: "<%= Name %>",
-		// TODO: Add commands here
-		Commands: []*plugin.Command{
-			// Example of a command
-			{
-				Use:   "<%= Name %>",
-				Short: "Explain what the command is doing...",
-				Long:  "Long description goes here...",
-				Flags: []*plugin.Flag{
-					{Name: "my-flag", Type: plugin.FlagTypeString, Usage: "my flag description"},
-				},
-				PlaceCommandUnder: "ignite",
-				// Examples of adding subcommands:
-				// Commands: []*plugin.Command{
-				// 	{Use: "add"},
-				// 	{Use: "list"},
-				// 	{Use: "delete"},
-				// },
-			},
-		},
-		// TODO: Add hooks here
-		Hooks: []*plugin.Hook{},
-		SharedHost: <%= SharedHost %>,
+		Name: "<%= Name %>",<%= if (SharedHost) { %>
+		SharedHost: true,<% } %>
+		Commands: cmd.GetCommands(),
 	}, nil
 }
 
-func (app) Execute(ctx context.Context, cmd *plugin.ExecutedCommand, api plugin.ClientAPI) error {
-	// TODO: write command execution here
-	fmt.Printf("Hello I'm the example-plugin plugin\n")
-	fmt.Printf("My executed command: %q\n", cmd.Path)
-	fmt.Printf("My args: %v\n", cmd.Args)
-
-	flags, err := cmd.NewFlags()
-	if err != nil {
-		return err
-	}
-
-	myFlag, _ := flags.GetString("my-flag")
```

Comment:
This should be documented or added again

Reply:
Does a dedicated example for both `flags` and `getChain` in the `apps` repository suffice?